src/site/SearchResultSet.php

```diff
@@ -0,0 +1,60 @@
+<?hh // strict
+
+namespace HHVM\UserDocumentation;
+
+final class SearchResultSet {
+    private Map<string, string> $classes;
+    private Map<string, string> $functions;
+    private Map<string, string> $interfaces;
+    private Map<string, string> $hhvm_guides;
+    private Map<string, string> $hack_guides;
+
+    public function __construct() {
+        $this->classes = Map {};
+        $this->functions = Map {};
+        $this->interfaces = Map {};
+        $this->hhvm_guides = Map {};
+        $this->hack_guides = Map {};
+    }
+
+    public function addAPIResult(string $type, string $name): void {
+        if ($type === "class") {
+            $this->classes[$name] = sprintf("/hack/reference/class/%s/", $name);
```

Comment:
doesn't need fixing for this, but FYI, I mean to add a URI builder - so something like `GuidePageController::getURIBuilder()->setStringParam('class', $name)->GetURI()` to avoid hardcoded paths